Automattic/jetpack
# Subscriptions: Track "Set up" button clicks

Part of https://github.com/Automattic/jetpack/issues/36958

## Proposed changes:

It adds a click tracking to the "Set up" newsletter payment plans button.

The Newsletter settings page view is already tracked.

Events related to configuring newsletter payment plans need to be added to the Calypso repo.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the Newsletter setting
* Click the "Set up" button
* Make sure the click event has been recorded
